src/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -37,6 +37,20 @@ internal static class ThreadPoolGlobals
         public static bool enableWorkerTracking;
 
         public static readonly ThreadPoolWorkQueue workQueue = new ThreadPoolWorkQueue();
+
+#if CORECLR
```

Comment:
Are these new "#if CORECLR"s needed?  ThreadPool.cs is not currently shared.

Reply:
Wasn't sure if they were needed for Mono and if having references to `IAsyncStateMachineBox` at the shared src level was problematic. Will remove.